src/SamplingSchemes/SamplingSchemes.jl

```diff
@@ -0,0 +1,38 @@
+module SamplingSchemes
+
+using ExtXYZ
+using Setfield
+using Unitful
+using DataFrames
+using Combinatorics
+using Random
+using Statistics
+
+using ..MonteCarloMoves
+using ..Potentials
+using ..AbstractWalkers
+using ..AbstractLiveSets
+using ..EnergyEval
+using ..FreeBirdIO
+using ..Hamiltonians
+
+export NestedSamplingParameters, LatticeNestedSamplingParameters
+export sort_by_energy!, nested_sampling_step!
+export nested_sampling_loop!
+export MCRoutine, MCRandomWalkMaxE, MCRandomWalkClone, MCDemonWalk, MixedMCRoutine, MCNewSample
+
+export exact_enumeration, nvt_monte_carlo, wang_landau
+
+abstract type SamplingParameters end
+
+include("NestedSampling.jl")
```

Comment:
@yangmr04, why is the filename formatting scheme different for `NestedSampling.jl` vs. `exact_enumeration.jl`, `nvt_monte_carlo.jl`, etc.

Reply:
Yup, they will be fixed later once they are in the dev branch.